docs/source/migration.mdx

```diff
@@ -1098,42 +1160,46 @@ export type CacheScope = 'PUBLIC' | 'PRIVATE';
 
 You can no longer type `CacheScope.Public` or `CacheScope.Private`. Instead, just use the string `'PUBLIC'` or `'PRIVATE'`. Values defined as `CacheScope` will only accept those two values, so any typos are still caught at compile time.
 
-
-
 ## Plugin API changes
 
+Apollo Server 4 makes many changes to the plugin API, including moving plugins from `apollo-server-core` to [deep imports within the `@apollo/server` package](#plugins-are-in-deep-imports). The below section covers Apollo Server 4's additional changes to the plugin API.
```

Comment:
I'm not sure I'd classify moving *specific plugins* around to fall under changes to "the plugin API".

Reply:
You make a good point! I mainly wanted to add a link to this section _just_ in case folks missed the opening section about plugins moving.

So we could do something like this:
```suggestion
Apollo Server 4 moves many plugins from `apollo-server-core` to [deep imports within the `@apollo/server` package](#plugins-are-in-deep-imports). The below section covers the changes that Apollo Server 4 makes to the plugin API.
```

Or we could remove this paragraph altogether; I think it reads fine either way we do it. What do you think @glasser?

Reply:
Just seems like it's about a different thing.

People confused about why they're having trouble importing ApolloServerPluginUsageReporting will search that string and find the right part quickly I think...

Reply:
Makes sense to me. I'll remove this paragraph and merge this baby! ✨

Reply:
```suggestion
```